highlight.io/components/QuickstartContent/self-host/dev-deploy.tsx

```diff
@@ -28,19 +28,19 @@ cd highlight/docker;
 ./start-infra.sh;`,
 					language: 'bash',
 				},
-				{
-					key: 'frontend',
-					text: `# now, start the frontend. this will run yarn docker:frontend in the monorepo and block until you stop it with ctrl+c.
-./run-frontend.sh`,
-					language: 'bash',
-				},
 				{
 					key: 'backend',
 					text: `# in a second terminal, start the backend. this will run make start-no-doppler in backend and block until you stop it with ctrl+c.
 cd highlight/docker;
 ./run-backend.sh`,
 					language: 'bash',
 				},
+				{
+					key: 'frontend',
+					text: `# now, start the frontend in a third terminal. this will run yarn docker:frontend in the monorepo and block until you stop it with ctrl+c.
```

Comment:
```suggestion
					text: `# now, start the frontend in a second terminal. this will run yarn docker:frontend in the monorepo and block until you stop it with ctrl+c.
```